packages/jira-adapter/src/filters/field_configuration_trashed_fields.ts

```diff
@@ -0,0 +1,31 @@
+/*
+*                      Copyright 2022 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+import { isInstanceElement, isReferenceExpression, Values } from '@salto-io/adapter-api'
+import { FilterCreator } from '../filter'
+
+const filter: FilterCreator = () => ({
+  onFetch: async elements => {
+    elements
+      .filter(isInstanceElement)
+      .filter(instance => instance.elemID.typeName === 'FieldConfiguration')
+      .forEach(instance => {
+        instance.value.fields = instance.value.fields
+          && instance.value.fields.filter((field: Values) => isReferenceExpression(field.id))
```

Comment:
1. nit: maybe
```suggestion
        if (instance.value.fields !== undefined) {
          instance.value.fields = instance.value.fields.filter((field: Values) => isReferenceExpression(field.id))
```
2. maybe we should log about fields that get omitted?